ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
+  ;; TODO fix an issue where it moves the cursor to the top after adding
+  (interactive)
+  (ebuku-update-tags-cache)
+  (let ((tag (completing-read "Append tag? " ebuku-tags))
+	(index (ebuku--get-index-at-point))
+	(inhibit-read-only t))
+    (ebuku--tags-add index tag)
+    (ebuku-refresh)
+    (message "Added tag to bookmark")))
+
+(defun ebuku--tags-join (tags)
+  "Internal function that joins a list of tags into a comma separated string of tags."
+  (typecase tags
```

Comment:
This is from `cl`, which i'd prefer to not have as a dependency. Please rewrite this to not make use of `cl-typecase`.

Reply:
Good catch, I didn't know that it was part of cl. Changed it to `cond`.